recipes-devtools/nodejs/nodejs_14.18.1.bb

```diff
@@ -0,0 +1,197 @@
+DESCRIPTION = "nodeJS Evented I/O for V8 JavaScript"
```

Comment:
Can you please leave a comment that this is backported version of the nodejs and provide a link where it's coming from?
Ofc if it's available on a separate meta-layer it's better to include that layer (but not OE one as upgrading this one mean upgrading to newer OE)